youtube_dl/extractor/flipagram.py

```diff
@@ -0,0 +1,105 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import time
+from .common import InfoExtractor
+
+from ..utils import (
+    parse_iso8601,
+    unified_strdate,
+)
+
+
+class FlipagramIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?flipagram\.com/f/(?P<id>[^/?_]+)'
+    _TESTS = [{
+        'url': 'https://flipagram.com/f/myrWjW9RJw',
+        'md5': '541988fb6c4c7c375215ea22a4a21841',
+        'info_dict': {
+            'id': 'myrWjW9RJw',
+            'title': 'Flipagram by crystaldolce featuring King and Lionheart by Of Monsters and Men',
+            'description': 'Herbie\'s first bannana🍌🐢🍌.  #animals #pets #reptile #tortoise #sulcata #tort #justatreat #snacktime #bannanas #rescuepets  #ofmonstersandmen  @animals',
+            'ext': 'mp4',
+            'uploader': 'Crystal Dolce',
+            'creator': 'Crystal Dolce',
+            'uploader_id': 'crystaldolce',
+        }
+    }, {
+        'url': 'https://flipagram.com/f/nyvTSJMKId',
+        'only_matching': True,
+    }]
+
+    def c_date_to_iso(self, c_date):
+        'Convert dates in format \'04/25/2016 00:23:24 UTC\' to ISO8601.'
+        return time.strftime('%Y-%m-%dT%H:%M:%S', time.strptime(c_date, '%m/%d/%Y %H:%M:%S %Z'))
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        self.report_extraction(video_id)
+        user_data = self._parse_json(self._search_regex(r'window.reactH2O\s*=\s*({.+});', webpage, video_id), video_id)
+        content_data = self._search_json_ld(webpage, video_id)
+
+        flipagram = user_data.get('flipagram', {})
+        counts = flipagram.get('counts', {})
+        user = flipagram.get('user', {})
+        video = flipagram.get('video', {})
+
+        thumbnails = []
+        for cover in flipagram.get('covers', []):
+            if not cover.get('url', ''):
+                continue
+            thumbnails.append({
+                'url': self._proto_relative_url(cover.get('url')),
+                'width': cover.get('width'),
```

Comment:
All integer fields must be wrapped in `int_or_none`, I've already mentioned that.